hedera-node/configuration/mainnet/bootstrap.properties

```diff
@@ -0,0 +1,21 @@
+# This file needs to exist until the transition from mono-service config to hedera-config is complete.
+# Downstream processes depend on its existence.
+#
+# This file MUST match application.properties exactly as they are both a replica of file 121 on mainnet and
+# this file is utilized by the mono-service config for property overrides whereas application.properties is utilized as
+# an override by the hedera-config base config.
+autoRenew.targetTypes=
+# Of historical interest only---block 33476932 was the "synchronization block" used to align the consensus
+# node block numbers with mirror node blocks in the 0.26 Services release
+contracts.knownBlockHash=055fac8eb788872957dfee45b7d40f7c94d66ea0a75ec703b993d57e441de574@33476932
+contracts.localCall.estRetBytes=4096
+hedera.allowances.isEnabled=true
+scheduling.whitelist=CryptoTransfer,ConsensusSubmitMessage,TokenBurn,TokenMint,CryptoApproveAllowance
+staking.fees.stakingRewardPercentage=10
+staking.maxStakeRewarded=650000000000000000
+staking.rewardBalanceThreshold=8500000000000000
+tokens.nfts.maxAllowedMints=10_000_000
+accounts.maxNumber=20_000_000
+#Overrides that differ based on the network
+ledger.id=0x01
+contracts.chainId=296
```

Comment:
Wrong values, or at least inconsistent with application.properties

Reply:
This means we missed it in develop also https://github.com/hashgraph/hedera-services/pull/9055/files#diff-a6ecad5c0ab8b2fe856e6591801581787a01e659e1a9d334b65b249423069c14R21

Reply:
Updating develop with this PR https://github.com/hashgraph/hedera-services/pull/9079